tests/tensorflow/helpers.py

```diff
@@ -20,13 +20,14 @@
 
 from nncf import NNCFConfig
 from nncf.tensorflow.helpers.model_creation import create_compressed_model
+from nncf.tensorflow.tf_internals import imagenet_utils, Rescaling # pylint: disable=unused-import
```

Comment:
Is it necessary?

Reply:
Right It was from miss importing from another file so it can be removed.